integrations/pjrt/CMakeLists.txt

```diff
@@ -38,7 +38,9 @@ if(IREE_PJRT_ENABLE_LTO)
 endif()
 
 # Customize defaults.
-option(IREE_BUILD_COMPILER "Disable compiler for runtime-library build" OFF)
+# IREE_BUILD_COMPILER should be enabled to make target IREELLVMIncludeSetup available,
+# which is required by PJRT dylib targets
+option(IREE_BUILD_COMPILER "Enable compiler for runtime-library build" ON)
 option(IREE_BUILD_SAMPLES "Disable samples for runtime-library build" OFF)
```

Comment:
Yeah, without IREELLVMIncludeSetup the linker will get abort with missing libs while building the PJRT dylibs. And seems that the IREE compiler C binding headers requires some mlir-c headers to be available (something like `MlirAttribute`, `MlirContext` appear in func sigs in the binding). I think maybe we can make IREELLVMIncludeSetup available regardless of whether IREE_BUILD_COMPILER is ON? since it's only a interface target composed of header dirs. After all maybe we can do this later : )